app/controllers/documents_controller.rb

```diff
@@ -37,7 +52,11 @@ def destroy
   private
 
   def permitted_params
-    params.require(:document).permit(:category, :file)
+    params.require(:document).permit(:category, :file, :file_name)
+  end
+
+  def update_transaction # removed the equals thing here, don't know if it still works
```

Comment:
This comment does not gives relevant information to the reader without context. It's better to delete it.
Btw, if you don't know if it still works, then you should write a test to check that.

Reply:
Ooops, my bad. Made the comment for myself but forgot to delete